packages/@webex/plugin-cc/src/features/Agent.ts

```diff
@@ -34,4 +34,25 @@ export default class Agent {
       return Promise.reject(new Error('Error while performing agent login', error));
     }
   }
+
+  public async getBuddyAgents(options: {
+    agentProfileId?: string;
+    mediaType: BuddyAgentMediaType;
+    state?: BuddyAgentState;
+  }): Promise<BuddyAgentsResponse> {
+    const {agentProfileId, mediaType, state} = options;
```

Comment:
Why are we fetching it here and then passing it one by one. We can pass the data as it is. We are not doing anything with the data herfe